### PR TITLE
Rename webview tag to element

### PR DESCRIPTION
**Pull request by miniak**
It's more correct to use the term element, as tag only refers to `<webview>` in HTML code. The DOM contains elements, also the API is `document.createElement` not `document.createTag`

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ an issue:
 ### Custom DOM Elements:
 
 * [`File` Object](api/file-object.md)
-* [`<webview>` Tag](api/webview-tag.md)
+* [`<webview>` Element](api/webview-element.md)
```
Comment (MarshallOfSound) [inline docs/README.md]: Worth noting this file name change will break historical links on the `electron.atom.io` website.

/cc @zeke do we have a handling mechanism for renames like this already somewhere?

**Comment by MarshallOfSound** [comment]
Tbh I'm not sure I completely agree with this change

* It results in a breaking API change (see the `BrowserWindow` constructor options)
* It is purely semantic
* And I personally prefer "tag" over "element" in some of the uses where it's been replaced in this PR.

When references instances of webviews it makes sense to call them elements, but for instance the option that enables webviews makes sense to be called `webviewTag` because you are enabling the tag itself.

Also we should be avoiding breaking changes 😄

**Comment by MarshallOfSound** [comment]
For context as to the usage of `tag` vs `element` please note the relationship between tag an element in your example of `document.createElement`, the first parameter of `document.createElement` is `tagName`.  I don't think it's a case of one or the other, just need to use the most appropriate one for each use case as described above

File: docs/README.md (hunk 1 above)
Comment (miniak) [inline docs/README.md]: I've deliberately left the old webview-tag.md file with a link to the new one to keep URLs working :)

File: atom/common/options_switches.cc [hunk 2]
```diff
@@ -182,7 +182,7 @@ const char kOpenerID[]         = "opener-id";
 const char kScrollBounce[]     = "scroll-bounce";
 const char kHiddenPage[]       = "hidden-page";
 const char kNativeWindowOpen[] = "native-window-open";
-const char kWebviewTag[]       = "webview-tag";
+const char kWebviewElement[]   = "webview-element";
```
Comment (miniak) [inline atom/common/options_switches.cc]: I am not sure if I can rename this, I haven't found the place parsing the command line option yet.

**Comment by zeke** [comment]
I tend to agree with @MarshallOfSound on this. It would mean a breaking change to the API, it might break things in unexpected ways, and is probably not worth the trouble for the slight semantic improvement.

**Comment by jkleinsc** [comment]
@miniak thanks for your PR and the effort you have put into it.  Given the consensus above, I am going to close this PR.

**Pull request closed without merging**